config/locales/en.yml

```diff
@@ -450,6 +451,7 @@ en:
     published: "This project has been published! Editors may enroll by visiting the following URL:"
     revisions_none: This project has no recent editing activity.
     school: Institution
+    school_and_term: Institution/When
```

Comment:
It's not great, but it matches the term we use for the 'term' field for courses that aren't actually classroom/academic projects.